### PR TITLE
Remove helm

### LINKED ISSUE #624: [Feature Request] Update knative canary with progressive rollout instructions to explicitly install RBAC rules for helm
**Is your feature request related to a problem? Please describe.**
Helm RBAC rules are currently installed as part of Iter8 install. However, they aren't needed for Iter8, only for particular use cases. Remove from Iter8 install and make part of the tutorial instead.

**Why is the feature useful for Iter8 users?**
This reduces the set of RBAC rules created for a basic Iter8 install. 

**Describe the solution you'd like**
Described above.

**Describe alternatives you've considered**
None.

**Does this issue require a design doc/discussion? If there is a link to the design document/discussions, please provide it below.**
You can use this issue thread, or a discussion thread in this repo, for design discussions. If you do so, please provide a link to the specific issue/discussion comment(s) that describes the design.

**How will this feature be tested?**
[ ] Tutorial still works. Test case for it works as well.

**How will this feature be documented?**
[ ] update tutorial

### PR DESCRIPTION
**Pull request by kalantar**
- Replaces https://github.com/iter8-tools/iter8/pull/618 which got polluted with changes from another change to master.
- Resolves https://github.com/iter8-tools/iter8/issues/624
- Updates version of iter8-install from 0.3.0 to 0.3.2
- Introduces environment variable GIT_ACCOUNT used by the install scripts. This can be set for manual testing against a fork.

### CODE REVIEW

File: samples/knative/quickstart/platformsetup.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ if [[ ! " ${NETWORK_LAYERS[@]} " =~ " ${1} " ]]; then
 fi
 
 # Step 1: Export correct tags for install artifacts
+export GIT_ACCOUNT="${GIT_ACCOUNT:-iter8-tools}"
 export TAG="${TAG:-v0.3.0}"
```
Comment (sriumcp) [inline samples/knative/quickstart/platformsetup.sh]: Tag needs to change here to v0.3.2

File: samples/knative/quickstart/platformsetup.sh [hunk 2]
```diff
@@ -114,12 +115,14 @@ fi
 ### Note: the preceding steps perform domain install; following steps perform Iter8 install
 
 # Step 5: Install Iter8
-echo "Installing Iter8"
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install.sh | bash
+echo "Installing Iter8 with Knative support"
+curl -s https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/install.sh | bash
```
Comment (sriumcp) [inline samples/knative/quickstart/platformsetup.sh]: ${TAG} needs to be replaced with  `main`

File: samples/knative/quickstart/platformsetup.sh [hunk 3]
```diff
@@ -114,12 +115,14 @@ fi
...
 
-# Step 6: Install Iter8's Prometheus add-on
+# Step 6: Install sample metrics for Knative and Iter8's Prometheus add-on
+echo "Installing sample metrics"
+kubectl apply -f https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/metrics/build.yaml
 echo "Installing Iter8's Prometheus add-on"
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install-prom-add-on.sh | bash
+curl -s https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/install-prom-add-on.sh | bash
```
Comment (sriumcp) [inline samples/knative/quickstart/platformsetup.sh]: ${TAG} needs  to change to `main`

File: samples/knative/quickstart/platformsetup.sh [hunk 4]
```diff
@@ -114,12 +115,14 @@ fi
...
 # Step 5: Install Iter8
-echo "Installing Iter8"
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install.sh | bash
+echo "Installing Iter8 with Knative support"
+curl -s https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/install.sh | bash
 
-# Step 6: Install Iter8's Prometheus add-on
+# Step 6: Install sample metrics for Knative and Iter8's Prometheus add-on
+echo "Installing sample metrics"
+kubectl apply -f https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/metrics/build.yaml
```
Comment (sriumcp) [inline samples/knative/quickstart/platformsetup.sh]: This ${TAG} is fine as it is.

File: mkdocs/docs/getting-started/install.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install
     set -e
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/install.md]: @kalantar If you agree, I am in favor of removing the ??? info "Look inside install.sh" tooltip altogether in  install.md. It is more work for us in terms of ensuring consistency/correctness of the documentation

File: mkdocs/docs/getting-started/install.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install
...
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/install.md]: @kalantar If you agree, I am in favor of removing the ??? info "Look inside instal-prom-add-on.sh" tooltip altogether in install.md. It is more work for us in terms of ensuring consistency/correctness of the documentation. Of course, folks can navigate to the GitHub page and look inside!

**Review by sriumcp — CHANGES_REQUESTED** [review]

File: samples/knative/quickstart/platformsetup.sh (hunk 2 above)
Comment (kalantar) [reply samples/knative/quickstart/platformsetup.sh]: @sriumcp I don't understand why this is the case. Why do we not want to install a particular version that the sample was written/tested against?

File: mkdocs/docs/getting-started/install.md (hunk 5 above)
Comment (kalantar) [reply mkdocs/docs/getting-started/install.md]: agree. done.

File: mkdocs/docs/getting-started/install.md (hunk 6 above)
Comment (kalantar) [reply mkdocs/docs/getting-started/install.md]: agree. done.

File: samples/knative/quickstart/platformsetup.sh (hunk 2 above)
Comment (sriumcp) [reply samples/knative/quickstart/platformsetup.sh]: What we want to ensure is that a specific version of `iter8` corresponds to a specific version of `iter8-install` (and `iter8ctl`).

Once you pin the version of `iter8-install`, by extension, you also pin the version of `handler`, `etc3` and `iter8-analytics`.

If each tutorial points to a different version of iter8-install, then we will pretty soon be staring at an unmanageable mess in terms of dependencies.

Comment (sriumcp) [reply samples/knative/quickstart/platformsetup.sh]: Re: testing, we do want to test the samples against the `iter8-install` version that install.md points to.

If the tutorials break for the version installed by `install.md`, we have a problem, and shouldn't be merging in the first place.

Comment (kalantar) [reply samples/knative/quickstart/platformsetup.sh]: ok. I think I follow the expected behavior. I have undone most of the changes in this script.

File: samples/knative/canaryprogressive/helm-rbac.yaml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  kind: ClusterRole
+  name: iter8-secret-reader
+subjects:
+- kind: ServiceAccount
+  name: iter8-handlers
+  namespace: iter8-system
+---
+# Helm v3 uses secrets to store Helm release information.
+# 
+# By default, when Helm is used within Iter8 experiments, secrets
```
Comment (sriumcp) [inline samples/knative/canaryprogressive/helm-rbac.yaml:36]: This is not true any longer.

File: samples/knative/canaryprogressive/helm-rbac.yaml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs: ["create", "update", "patch", "delete"]
+---
+# Helm v3 uses secrets to store Helm release information.
+# 
+# By default, when Helm is used within Iter8 experiments, secrets 
+# will be located in the Iter8 namespace.
```
Comment (sriumcp) [inline samples/knative/canaryprogressive/helm-rbac.yaml]: No longer true.

File: samples/knative/canaryprogressive/helm-rbac.yaml [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+#
+# This role binding enables Iter8 handler to read
```
Comment (sriumcp) [inline samples/knative/canaryprogressive/helm-rbac.yaml:19]: Default namespace

File: samples/knative/canaryprogressive/helm-rbac.yaml [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+  name: iter8-handlers
+  namespace: iter8-system
+---
+# Helm v3 uses secrets to store Helm release information.
+# 
+# By default, when Helm is used within Iter8 experiments, secrets 
+# will be located in the Iter8 namespace.
+#
+# This role binding enables Iter8 handler to write 
+# secrets in the namespace where Iter8 is installed
```
Comment (sriumcp) [inline samples/knative/canaryprogressive/helm-rbac.yaml]: default  namespace

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor fixes

File: mkdocs/docs/tutorials/knative/canary-progressive.md [hunk 11]
```diff
@@ -98,7 +106,7 @@ sed "s+URL_VALUE+${URL_VALUE}+g" $ITER8/samples/knative/canaryprogressive/fortio
           restartPolicy: Never    
     ```
 
-## 3. Create Iter8 experiment
+## 4. Create Iter8 experiment
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/knative/canary-progressive.md:109]: When you `look inside` the Iter8-experiment, it still says `--namespace=iter8-system`. The new experiment  should be copied in here...

**Review by sriumcp — CHANGES_REQUESTED** [review]

File: mkdocs/docs/tutorials/knative/canary-progressive.md [hunk 12]
```diff
@@ -203,11 +211,11 @@ Observe the experiment in realtime. Paste commands from the tabs below in separa
...
-## 5. Cleanup
+## 6. Cleanup
 ```shell
 kubectl delete -f $ITER8/samples/knative/canaryprogressive/experiment.yaml
 kubectl delete -f $ITER8/samples/knative/canaryprogressive/fortio.yaml
-helm uninstall sample-app --namespace=iter8-system
+helm uninstall sample-app --namespace=default
 ```
 
 ???+ info "Understanding what happened"
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/knative/canary-progressive.md:221]: Understanding what happened also needs to change -- `iter8-system` must become `default`

**Review by sriumcp — CHANGES_REQUESTED** [review]

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**